utils/systemd/monero.service

```diff
@@ -0,0 +1,88 @@
+### Sample unit file for running monerod as a systemd service ###
+
+### For more information about the options specified in the [Unit] and [Install] sections,
+### please see the systemd.unit(5) man page.
+
+[Unit]
+
+Description=Monero Daemon
+After=network.target
+
+### If monerod is configured to use Tor via "--tx-proxy" and, optionally, "--anonymous-inbound",
+### and if Tor is installed and running as a systemd service,
+### then replace the above "After=network.target" directive with the following two directives:
+
+# After=network.target tor.service
+# Wants=tor.service
+
+### The following [Install] section allows monero.service to be enabled via "systemctl enable",
+### which in turn causes monero.service to be automatically started at system startup.
+### The "Alias=monerod.service" directive allows monero.service to be invoked as monerod.service in various systemctl commands
+### if monero.service is enabled via "systemctl enable".
+
+[Install]
+
+Alias=monerod.service
+WantedBy=multi-user.target
+
+### For more information about the options specified in the [Service] section,
+### please see the systemd.service(5) and systemd.exec(5) man pages.
+
+[Service]
+
+Type=simple
+
+### The following "ExecStart=" line assumes that:
+### (a) the monerod binary has been copied to the /usr/local/bin directory and is owned by the "monero" system user and group,
+### (b) the /etc/monero directory exists and is owned by the "monero" system user and group, and
+### (c) the monerod.conf configuration file exists, is located in /etc/monero, and is owned by the "monero" system user and group.
+
+ExecStart=/usr/local/bin/monerod --config-file /etc/monero/monerod.conf --non-interactive
+ExecStop=/usr/local/bin/monerod exit
```

Comment:
This shouldn't be needed.

Reply:
If you mean that the `ExecStop=/usr/local/bin/monerod exit` directive isn't needed, I've removed that directive in commit `cb5b8f5`.